app/javascript/packages/i18n/package.json

```diff
@@ -1,5 +1,28 @@
 {
   "name": "@18f/identity-i18n",
-  "private": true,
-  "version": "1.0.0"
+  "private": false,
+  "version": "1.0.0",
```

Comment:
@allthesignals  Wondering if I should bump the version to `1.0.1` or if you can delete version 1.0.0 in npm?

Reply:
NPM versions can't be deleted, so I think you're going to have to bump it if you want to publish a new version.

Reply:
I think the [beta versions](https://www.npmjs.com/package/@18f/identity-i18n?activeTab=versions) were a good idea and we should use that moving forward for test versions. We should also make sure that they're actually [tagged as beta](https://docs.npmjs.com/cli/v8/commands/npm-publish#tag).

Reply:
Thanks. I didn't think they could be deleted- or at least not best practice to delete.
Version bumped up, resolved with commit 360929ac4c5cabcb9828de5404cb3ee0f226e003